fdm-core/src/cultivation.ts

```
@@ -719,6 +771,24 @@
 }
 
+const isCultivationWithinTimeframe = (
+    b_lu_start: Date | null,
+    b_lu_end: Date | null,
+    timeframe: Timeframe,
+): boolean => {
+    if (!b_lu_start) return false
+
+    if (b_lu_end) {
+        return (
+            (b_lu_start >= timeframe.start! && b_lu_start <= timeframe.end!) ||
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)